external-crates/move/crates/move-analyzer/trace-debug/src/extension.ts

```diff
@@ -685,8 +685,11 @@ function splitTraceFileLines(decompressed: Uint8Array): string[] {
  * @returns decompressed trace file content as a string.
  */
 async function decompressTraceFile(traceFilePath: string): Promise<string[]> {
+    // Read as Buffer then view it as a Uint8Array for fzstd which expects Uint8Array
     const buf = fs.readFileSync(traceFilePath);
-    const decompressed = await decompress(buf);
+    const u8 = new Uint8Array(buf.buffer, buf.byteOffset, buf.byteLength);
```

Comment:
Another unrelated fix (for some reason the code was actually working even though TS compiler didn't like the types - now compilation is clean again)